org.mozilla.FirefoxNightly.flashplayer/org.mozilla.FirefoxNightly.flashplayer.json

```diff
@@ -0,0 +1,50 @@
+{
+    "id": "org.mozilla.FirefoxNightly.flashplayer",
+    "runtime": "org.mozilla.FirefoxNightly",
+    "build-extension": true,
+    "sdk": "org.gnome.Sdk//3.26",
+    "separate-locales": false,
+    "appstream-compose": false,
+    "build-options": {
+	"prefix": "/app/lib/mozilla/plugins/",
+	"env": {
+	    "PATH": "/app/lib/mozilla/plugins/bin:/app/bin:/usr/bin"
+        }
+    },
+    "modules": [
+        {	
+	    "name": "flash_player",
+            "buildsystem": "simple",
+            "build-commands": [
+		"mkdir -p flash_player",
+	        "tar xvzf flash_player.tar.gz -C flash_player",
+		"cp flash_player/libflashplayer.so /app/lib/mozilla/plugins",
+		"rm -rf flash_player flash_player.tar.gz"
+	    ],
+	    "sources": [
+                {
+                    "type": "file",
```

Comment:
This should not use file - then we are mirroring a copy of the plugin. Better to use extra-data, then we would move the build-commands into a simple apply_extra.sh script - then the plugin will be downloaded on the client side, the checksum verified, and then unpacked into the local deploy of the Flatpak. The apply commands then just move things into place. https://github.com/flathub/us.zoom.Zoom is a nice simple example.